samples/snippets/authorized_view_tutorial_test.py

```diff
@@ -38,7 +39,7 @@ def test_authorized_view_tutorial(
     client: bigquery.Client, datasets_to_delete: List[str]
 ) -> None:
     override_values = {
-        "source_dataset_id": "github_source_data_{}".format(
+        "source_dataset_id": f"{prefixer.create_prefix()}_authorized_view_tutorial".format(
             str(uuid.uuid4()).replace("-", "_")
         ),
```

Comment:
We should be able to rely upon `prefixer.create_prefix` to create a value for these datasets.
The value will start with something the looks like the `repo` name and the `folder directory` (`self._prefix`).

The cleanup script will look for any datasets that `.startswith()` the value stored as `self._prefix`.

```suggestion
        "source_dataset_id": f"{prefixer.create_prefix()}_authorized_view_tutorial",
```
We don't need to create a universally unique identifier (using `uuid`), since the `prefixer.create_prefix()` function already does that using a function from the `random` library and the `hex()` function. Also `prefixer.create_prefix()` doesn't inadvertantly inject any undesirable separators, so the `replace()` function is not needed.

Note: there are additional places in this code that create dataset_ids, such as the `shared_dataset_id` below. 

Please confirm whether they will also need to be deleted.